Configurateur/front/lib/screens/container-creation/maps_screen/maps_screen.dart

```diff
@@ -191,6 +190,37 @@ class MapsState extends State<MapsScreen> {
               ],
             ),
           ),
+          const SizedBox(
+            height: 50,
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              ProgressBar(
+                length: 6,
+                progress: 4,
+                previous: AppLocalizations.of(context)!.previous,
+                next: AppLocalizations.of(context)!.next,
+                previousFunc: goPrevious,
+                nextFunc: goNext,
+              ),
+            ],
+          ),
+          Center(
```

Comment:
bad position for the Icon, put it in the top-right or top-left corner of the maps but not in the center

Reply:
look where he is positionned right now on the video, it's the best position and I can't put it just above the map so it's the best place